api-test/pom.xml

```diff
@@ -54,9 +54,9 @@
 
 	<dependencies>
 		<dependency>
-			<groupId>io.mosip.testrig.apirig.apitest.commons</groupId>
+			<groupId>io.mosip.testrig.apitest.commons</groupId>
 			<artifactId>apitest-commons</artifactId>
-			<version>1.2.2-SNAPSHOT</version>
+			<version>1.3.0</version>
```

Comment:
@nandhu-kumar 1.3.0 is java21 and esignet still not migrated to java21.
it will work ?

Reply:
our API test rig already migrated to java 21.... so there won't be any conflicts

Reply:
core module not migrate to java21

Reply:
There is no link between the core component and our test rig in the form of a Java version. so there won't be any issue.